src/views/home.vue

```diff
@@ -1,9 +1,11 @@
 <template>
   <div class="container">
     <div class="Home">
-      <h1> Jorge Adanza</h1>
-      <p>June 28, 2024</p>
-      <button type="button" class="btn btn-outline-success"><RouterLink to="/app">Go To App</RouterLink></button>
+      <h1>Jorge Adanza</h1>
+      <p>July 5, 2024</p>
+      <button type="button" class="btn btn-outline-success">
+        <RouterLink to="/app">Go To App</RouterLink>
```

Comment:
use the named route
`<RouterLink to={name:routeNames.todo.path} />`

Reply:
acknowledged